pkg/install/install.go

```diff
@@ -234,7 +234,7 @@ func (i *Installer) runSteps(ctx context.Context, steps []interface{}) error {
 					err = step(ctx)
 					if azureerrors.HasAuthorizationFailedError(err) ||
 						azureerrors.HasLinkedAuthorizationFailedError(err) {
-						i.log.Print(err)
+						i.log.Warnf("found link authorization errors, will retry (%s)", err)
```

Comment:
This is intentional. The print is PRIORITY 6 error, so by checking priority you know it is retriable. This is pattern in all codebase. Warning, on the other hand, is Priority 4. 
https://golang.org/src/log/syslog/syslog.go#L24

Reply:
I know Print is level 6, but I think exceptions that are logged but are not considered errors should be Warnings, WARN is log level 4 like you point out. I am changing it back to Print but I still think logging exceptions with Info level is a logging anti-pattern